### PR TITLE
Add small success rate chart to table, misc web tweaks

### LINKED ISSUE #1589: Add little success rate chart to metrics tables
Add a small success rate chart to the Metrics tables:

screen shot 2018-09-05 at 3 10 50 pm

### LINKED ISSUE #1598: Rename upstream/downstream to inbound/outbound in UI
Change the displayed words.

### LINKED ISSUE #1600: Adjust displayed precision on latency data
We should display more dp if the data is at the second level.

From William:
> latency table was showing things like "2.94ms" but also "3s". it should show "3.06s" or something with more degrees of precisi9on

### LINKED ISSUE #1610: Assign widths to metrics table columns so that tables align
There's a lot of places where we display the same tables for different resources, and even though the columns are the same, they don't align across resources because we're not setting any widths.

We should add fixed widths to the columns.
Also note we'd need to figure out how these fixed width columns look on a widescreen view.

screen shot 2018-09-07 at 11 03 58 am
screen shot 2018-09-07 at 11 05 51 am

### PR DESCRIPTION
**Pull request by rmars**
- Add a small success rate chart to the metrics tables
- Improve latency formatting for seconds latencies
- Rename upstream/downstream to inbound/outbound
- Make Top table look consistent with rest of tables on page
- Fix widths of metrics column columns so that tables align

The success rate mini chart is last in the Top table, but near first in the metric table. I felt like it might be good to put it farther away from the other coloured column (Grafana link), but the column order can easily be changed.

Something to think about: the VotePoop endpoint is showing up as grey (0% SR). You'd expect it to be red but because the gauge is also reflecting the actual percent, the coloured part of this is 0, so the red part doesn't show. An alternative is to actually just show a full arc no matter the value of the success rate. We could also colour the rows or text instead of adding this arc.

screen shot 2018-09-11 at 1 51 50 pm
screen shot 2018-09-11 at 2 04 24 pm

Fixes #1598 
Fixes #1600 
Fixes #1589
Fixes #1610

### CODE REVIEW

**Comment by rmars** [comment]
> is it possible to format these values so we always have 3 sig-digs, regardless of unit?

I think we should find the right mix. It'll be jarring to have like... 615µs and 10.3ms... the idea was just so that at the second level, we don't lose a bunch of granularity. But I feel like ms and µs are easier to comprehend at the non-decimal granularity. But if we format all the numbers to be non-decimal, we lose some info at the second display between 1 and 10 seconds (hence this change).

**Comment by rmars** [comment]
The Authorities not being aligned would actually be fixed by https://github.com/linkerd/linkerd2/issues/1225 so I'm inclined to just leave it as is until we can get to that ticket? Or if we urgently want it to be aligned, I think we should just add the Authorities in Grafana.

File: web/app/js/components/MetricsTable.jsx [hunk 1]
```diff
@@ -117,14 +119,16 @@ const columnDefinitions = (resource, namespaces, onFilterClick, showNamespaceCol
       dataIndex: "successRate",
       key: "successRateRollup",
       className: "numeric",
+      width: "120px",
```
Comment (klingerf) [inline web/app/js/components/MetricsTable.jsx]: This width needs to be slightly wider for when the sortable header is present:

image

Testing locally it looks like 130px would work.

Comment (rmars) [reply web/app/js/components/MetricsTable.jsx]: (note that this applies to the definition in `TopEventTable.jsx` and not the one in `MetricsTable`

**Pull request merged**